### PR TITLE
Doc and template changes from deploying additional spokes for pre-prod

### PR DESCRIPTION
**Pull request by AAkindele**
# Type of PR

- [x] Documentation changes
- [ ] Code changes
- [ ] Test changes
- [ ] CI-CD changes
- [ ] GitHub Template changes

## PR Checklist

- [x] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [ ] My code follows the code style of this project.
- [ ] I ran lint checks locally prior to submission.
- [x] Have you checked to ensure there aren't other open Pull Requests for the same update/change?

## Purpose of PR

Documentation updates from deploying additional spokes to the preprod environment. Most of these files are auto generated from templates. I have commented on the files that were manual changes.

- https://ngsa-memory-centralus-pre.cse.ms/index.html
- https://ngsa-cosmos-centralus-pre.cse.ms/index.html
- https://ngsa-java-centralus-pre.cse.ms/index.html - currently down. note in triage
- https://grafana-centralus-pre.cse.ms/

## Does this introduce a breaking change

- [ ] YES
- [x] NO

## Validation

- [ ] Unit tests updated and ran successfully
- [x] Update documentation or issue referenced above

## Issues Closed or Referenced

- References https://github.com/retaildevcrews/wcnp/issues/573

### CODE REVIEW

File: docs/deployAdditionalCluster.md [hunk 1]
```diff
@@ -64,10 +64,10 @@ export ASB_HUB_TUNNEL_FRONT_UDP_RULE_NAME=tunnelfront-pod-udp
 export ASB_HUB_POD_TO_API_SERVER_RULE_NAME=pod-to-api-server
```
Comment (AAkindele) [inline docs/deployAdditionalCluster.md:64]: [review note] manual changes

File: cluster/cluster-stamp-additional.json [hunk 2]
```diff
@@ -111,35 +111,6 @@
             "metadata": {
```
Comment (AAkindele) [inline cluster/cluster-stamp-additional.json:111]: [review note] manual changes

File: templates/fluentbit/config-log.yaml [hunk 3]
```diff
@@ -5,8 +5,8 @@ metadata:
   name: fluentbit-log-config
```
Comment (AAkindele) [inline templates/fluentbit/config-log.yaml:5]: [review note] manual changes

File: templates/loderunner.yaml [hunk 4]
```diff
@@ -1,71 +1,71 @@
-apiVersion: v1
```
Comment (AAkindele) [inline templates/loderunner.yaml:1]: [review note] manual changes. replaced CRLF with LF

**Review by joaquinrz — APPROVED** [review]

**Pull request merged**